packages/user-mgnt/src/features/resstPassword/handler.ts

```diff
@@ -39,6 +40,18 @@ export default async function resetPasswordSMSHandler(
     throw unauthorized()
   }
 
+  const remoteAddress =
+    request.headers['x-real-ip'] || request.info.remoteAddress
+  const userAgent =
+    request.headers['x-real-user-agent'] || request.headers['user-agent']
+
+  await postUserActionToMetrics(
+    'SEND_PASSWORD',
+    request.headers.authorization,
+    remoteAddress,
+    userAgent
+  )
+
```

Comment:
@favour234 similar to the ticket that you are working on, let's wrap the `postUserActionToMetrics` function call in a try catch block and use `logger.error` in catch block

Reply:
alright..would update